reti-ui-common/tsconfig.json

```diff
@@ -16,9 +16,10 @@
         "isolatedModules": true,
         "moduleDetection": "force",
         "strict": true,
-        "lib": ["ESNext", "DOM"]
+        "lib": ["ESNext", "DOM"],
+        "outDir": "./dist"
```

Comment:
Maybe remove this? Since typescript shouldn't generate any output, we let Vite do that.